app-backend/db/src/main/scala/ProjectDao.scala

```diff
@@ -84,44 +84,64 @@ object ProjectDao
-        extras)
-      VALUES
-        ($id, $now, $now, ${user.id},
-        ${user.id}, $ownerId, ${newProject.name}, $slug, ${newProject.description},
-        ${newProject.visibility}, ${newProject.tileVisibility}, ${newProject.isAOIProject},
-        ${newProject.aoiCadenceMillis}, $now, ${newProject.tags}, null,
-        TRUE, ${newProject.isSingleBand}, ${newProject.singleBandOptions}, null,
-        ${newProject.extras}
+    for {
+      project <- (fr"INSERT INTO" ++ tableF ++ fr"""
+          (id, created_at, modified_at, created_by,
+          modified_by, owner, name, slug_label, description,
+          visibility, tile_visibility, is_aoi_project,
+          aoi_cadence_millis, aois_last_checked, tags, extent,
+          manual_order, is_single_band, single_band_options, default_annotation_group,
+          extras)
+        VALUES
+          ($id, $now, $now, ${user.id},
+          ${user.id}, $ownerId, ${newProject.name}, $slug, ${newProject.description},
+          ${newProject.visibility}, ${newProject.tileVisibility}, ${newProject.isAOIProject},
+          ${newProject.aoiCadenceMillis}, $now, ${newProject.tags}, null,
+          TRUE, ${newProject.isSingleBand}, ${newProject.singleBandOptions}, null,
+          ${newProject.extras}
+        )
+      """).update.withUniqueGeneratedKeys[Project](
+        "id",
+        "created_at",
+        "modified_at",
+        "created_by",
+        "modified_by",
+        "owner",
+        "name",
+        "slug_label",
+        "description",
+        "visibility",
+        "tile_visibility",
+        "is_aoi_project",
+        "aoi_cadence_millis",
+        "aois_last_checked",
+        "tags",
+        "extent",
+        "manual_order",
+        "is_single_band",
+        "single_band_options",
+        "default_annotation_group",
+        "extras",
+        "default_layer"
       )
-    """).update.withUniqueGeneratedKeys[Project](
-      "id",
-      "created_at",
-      "modified_at",
-      "created_by",
-      "modified_by",
-      "owner",
-      "name",
-      "slug_label",
-      "description",
-      "visibility",
-      "tile_visibility",
-      "is_aoi_project",
-      "aoi_cadence_millis",
-      "aois_last_checked",
-      "tags",
-      "extent",
-      "manual_order",
-      "is_single_band",
-      "single_band_options",
-      "default_annotation_group",
-      "extras"
-    )
+      defaultProjectLayer <- ProjectLayerDao.insertProjectLayer(
+        ProjectLayer(UUID.randomUUID(),
+                     now,
+                     now,
+                     "default_layer",
+                     id,
+                     "#FFFFFF",
+                     None,
+                     None,
+                     None,
+                     None)
+      )
+      _ <- this.updateProject(
+        project.copy(defaultLayer = Some(defaultProjectLayer.id)),
+        id,
+        user)
+      dbProject <- unsafeGetProjectById(id)
```

Comment:
I don't think this is necessary
```
...
updatedProject = project.copy(defaultLayer = Some(defaultProjectLayer.id), id, user)
_ <- this.updateProject(updatedProject)
} yield updatedProject
```
the modified time could be slightly different but that doesn't seem like it's worth an extra query to get right.